contracts/staking/PoSValidatorManager.sol

```diff
@@ -191,72 +192,73 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         emit DelegatorAdded({
             validationID: validationID,
             setWeightMessageID: messageID,
-            delegator: delegator,
+            delegationID: delegationID,
```

Comment:
We should include the delegator address in the event, so that the delegation ID can be related to an address